### PR TITLE
New option to exclude available resolution separately

### PR DESCRIPTION
**Pull request by nuschk**

### CODE REVIEW

File: fingerprint2.js [hunk 1]
```diff
@@ -144,11 +144,13 @@
       if(!this.options.excludeScreenResolution) {
         return this.getScreenResolution(keys);
       }
+      if (!this.options.excludeAvailableResolution) {
```
Comment (Valve) [inline fingerprint2.js]: please call this excludeAvailableScreenResolution to be consistent

**Comment by Valve** [comment]
@nuschk did you get a chance to finalize this PR?

**Comment by nuschk** [comment]
@Valve I've changed the naming as you suggested. I think the PR is fine. Although, I must admit, I haven't really dug into the code too much. I didn't find anything to add to the documentation nor any tests, I hope that's correct.

**Comment by Valve** [comment]
@nuschk thanks for the update. Please add a spec to cover this (similar to this: https://github.com/Valve/fingerprintjs2/blob/master/specs/specs.js#L74) and we should be good to go

**Comment by nuschk** [comment]
@Valve Oops, I missed that. Will do. Thanks!

**Comment by nuschk** [comment]
@Valve So, I've added the spec and also made the available screen res independent from the "normal" screen res. I hope that somehow fits the architecture you had in mind.

**Comment by Valve** [comment]
thanks, released v.1.1.0

**Comment by nuschk** [comment]
Thanks! :+1:

**Pull request merged**